lib/modules/manager/leiningen/__snapshots__/extract.spec.ts.snap

```diff
@@ -1,5 +1,265 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`extractPackageFile 1`] = `
```

Comment:
I'm not sure why this export was added. I'm not very familiar with Jest's snapshotting behaviour, so I can remove if it seems like it shouldn't be here.

Reply:
I think this was an erroneuous addition during development. After running [pnpm jest -u lib/modules/manager/leiningen/extract.spec.ts](https://github.com/renovatebot/renovate/pull/29552/commits/fbd74135107540a8d03dc47d149a1aa2e6d8ae52) it's disappeared.